virttest/utils_libvirt/libvirt_monitor.py

```diff
@@ -53,7 +54,55 @@ def _check_items(value_type, tmp_domjobinfo):
     return tmp_domjobinfo
 
 
-def check_domjobinfo_output(params):
+def _get_value(expect_key, out):
+    """
+    Get value from out
+
+    :param expect_key: expect key
+    :param out: domain job info
+    :retrun: the value of expect key
+    """
+    for line in out.splitlines():
+        key = line.split(':')[0]
+        if key == expect_key:
+            value, unit = line.strip().split(':')[1].strip().split(' ')
+            if unit == "MiB":
+                value = round(float(value) / 1024, 3)
+            return float(value)
+    return None
+
+
+def _check_sum_items(line, key, tmp_domjobinfo, domjobinfo_out):
+    """
+    Check the items that need to be summed
+
+    :param line: one line of domjobinfo output
+    :param key: one key of domjobinfo output
+    :param tmp_domjobinfo: expect domain job info
```

Comment:
I think this function needs a special tmp_domjobinfo value to calculate the sum of some values, can you give an example in docstring so others can understand it clearly?

Reply:
Updated, thanks